bigquery/gcloud/aio/bigquery/bigquery.py

```diff
@@ -233,16 +268,21 @@ def _make_load_body(
         project = await self.project()
         url = f'{API_ROOT}/projects/{project}/jobs'
 
-        body = self._make_load_body(source_uris, project)
-        payload = json.dumps(body).encode('utf-8')
+        body = self._make_load_body(
+            source_uris, project, autodetect, source_format, write_disposition,
+        )
+        return await self._post_json(url, body, session, timeout)
 
-        headers = await self.headers()
-        headers.update({
-            'Content-Length': str(len(payload)),
-            'Content-Type': 'application/json',
-        })
+    async def query(
```

Comment:
I think this might get a bit confusing, since this is not a "bigquery query" but rather a "bigquery insert via query" -- looks like I didn't RTF very well when picking names. Could you please rename `load` to `insert_via_load`, `copy` to `insert_via_copy`, and this method to `insert_via_query`? Open to other naming schemes, of course, just want to make sure this doesn't confuse folks who are looking for the ability to actually run a query!

Reply:
If you could please also add the following comment above this method definition, that'd help us track things down in the future:
```
# https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs/insert
# https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationQuery
```